cmd/kubeadm/app/phases/copycerts/testutil_umask_noop.go

```diff
@@ -0,0 +1,23 @@
+// +build !darwin,!linux
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package copycerts
+
+func setNoUmask() {
```

Comment:
Like i've mentioned on the issue these file mask unit tests are broken on windows, since the OS simply does not have mode bits for files. Some of the go stdlib logic is no-op on windows. I think we cross compile the kubeadm binary for windows, but we never run unit tests on windows.
If we do, this change is accurate and probably more tests would fail in cmd/kubeadm.

Thanks
/approve
/triage accepted
/priority backlog
@pacoxu please apply LGTM given you were already reviewing.

Reply:
Yeah, I think maybe some limited windows unit test running occurs, but not in CI or the release and I would be surprised if it didn't have many issues similar to https://github.com/kubernetes/kubernetes/issues/102607 already ...